RELEASE_NOTES.md

```diff
@@ -77,8 +77,14 @@ CSM 1.4 contains approximately 500 changes spanning bug fixes, new feature devel
 
 ## Deprecations
 
+The following features are now deprecated and will be removed from CSM in a future release.
+
 * CAPMC v1 partial deprecation
 * HSM v1 interface
+* [BOS](glossary.md#boot-orchestration-service-bos) v1 is now deprecated, in favor of BOS v2. BOS v1 will be removed from CSM in the CSM-1.9 release.
+  * It is likely that even prior to BOS v1 being removed from CSM, the [Cray CLI](glossary.md#cray-cli-cray) will change its behavior when no
+    version is explicitly specified in BOS commands. Currently it defaults to BOS v1, but it may change to default to BOS v2 even before BOS v1
+    is removed from CSM.
```

Comment:
Should we call out how customers can try the "v2" API with the "cray" command?   e.g. `cray bos v2 sessiontemplates list`

Reply:
We do that in the BOS V2 section of the documentation.